test/vstest.console.UnitTests/Processors/RunSettingsArgumentProcessorTests.cs

```diff
@@ -363,6 +363,30 @@ public void InitializeShouldNotSetInIsolataionToTrueIfEnvironmentVariablesNotSpe
             Assert.IsNull(this.settingsProvider.QueryRunSettingsNode(InIsolationArgumentExecutor.RunSettingsPath));
         }
 
+        [TestMethod]
+        public void InitializeShouldUpdateTestCaseFilterIfProvided()
+        {
+            // Arrange.
+            var fileName = "C:\\temp\\r.runsettings";
```

Comment:
nit: Maybe those should be const string instead of var ?

Reply:
BTW why `C:\\temp` ? Why not something like:
`string fileName = Path.Combine(Path.GetTempPath(), "r.runsettings");`

Reply:
@cvpoienaru technically maybe, but this is not javascript, substituting var / let with const is not what most C# would use 🙂  
@pavelhorak this is basically a setter test, we don't touch the filesystem. So we don't need to query the underlying system to get temp. Any path here would be okay. 

Also I am simply following the style of the code around, consistent style is more important than forcible use of new patterns. 🙂

Reply:
@nohwnd, wouldn't know, never touched javascript anyways :)
To clarify, my background is C++ and now C#.

Reply:
Yeah I know :)